pkg/controllers/managementuser/rbac/handler_base.go

```diff
@@ -138,13 +138,13 @@ func Register(ctx context.Context, workload *config.UserContext) error {
 	management.Management.GlobalRoleBindings("").AddHandler(ctx, grbHandlerName, newGlobalRoleBindingHandler(workload))
 
 	sync := &resourcequota.SyncController{
-		Namespaces:          workload.Core.Namespaces(""),
 		NsIndexer:           nsInformer.GetIndexer(),
+		Namespaces:          workload.Management.Wrangler.Core.Namespace(),
+		ProjectGetter:       workload.Management.Wrangler.Mgmt.Project(),
```

Comment:
If this is only for getting projects, we can do
```suggestion
		ProjectCache:       workload.Management.Wrangler.Mgmt.Project().Cache(),
```